server/game/cards/01_SOR/units/DisablingFangFighter.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class DisablingFangFighter extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '2668056720',
+            internalName: 'disabling-fang-fighter'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addWhenPlayedAbility({
+            title: 'You may defeat an upgrade.',
```

Comment:
we do not include 'you may' on title, prompt menu handle it for us

Reply:
Gotcha, fixed